src/Controller/ExpositionController.java

```diff
@@ -0,0 +1,253 @@
+                Date endingDate = dateConfiguration();
+                if (startingDate.after(endingDate)) {
+                    System.out.println("Error! Starting date can not be after ending date.");
+                } else {
+                    expo.setStartDate(startingDate);
+                    expo.setEndDate(endingDate);
+                    updateExpoStatus(expo.getExpoName());
+                    System.out.println("Success! Exposition " + expoName + " created.");
+                    done = true;
+                }
+            }
+        } else {
+            System.out.println("Name already in use. Try again");
+            createExpo();
+        }
+
+    }
+
+    /**
+     * Method for the deletion of an Exposition
+     *
+     * @see #showExpositions()
+     * @see #updateExpoStatus(String)
+     */
+    public void deleteExpo() {
+        Scanner scanner = new Scanner(System.in);
+        showExpositions();
+        System.out.println("Exposition name to delete: ");
+        String expoName = scanner.next();
+        if (expoList.containsKey(expoName.toLowerCase())) {
+            updateExpoStatus(expoName.toLowerCase());
+            if(!expoList.get(expoName.toLowerCase()).getExpoStatus()) {
+                expoList.remove(expoName.toLowerCase());
+                System.out.println("Exposition " + expoName.toLowerCase() + " has been deleted.");
+            } else {
+                System.out.println("Exposition is Active. Can not be Deleted.");
+            }
+        } else {
+            System.out.println("Exposition not found. Please type the exact name");
+            deleteExpo();
+        }
+    }
+
+    /**
+     * Singleton for the ExpositionController
+     * @return The same ExpositionController instance
+     */
+    public static ExpositionController getInstance() {
+        if (ExpoListInstance == null) {
+            ExpoListInstance = new ExpositionController();
+        }
+        return ExpoListInstance;
+    }
+
+    /**
+     * Method to access the expoList
+     * @return expoList
+     */
+    public LinkedHashMap<String, Expositon> getExpoList() {
+        return expoList;
+    }
+
+    /**
+     * Method to print all the Expositions in expoList
+     */
+    public void showExpositions() {
+        LinkedHashMap<String, Expositon> invList = expoList;
+        if (!expoList.isEmpty()) {
+            Set<String> expoKey = invList.keySet();
+            int index = 1;
+            for (String exposition : expoKey) {
+                System.out.println(index + ". " + exposition);
+                index++;
+            }
+        } else {
+            System.out.println("Exposition List is empty");
+        }
+    }
+
+    /**
+     * Method to configure a Date
+     *
+     * @return Date
+     */
+    private Date dateConfiguration() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Select Year");
+        int startYear = scanner.nextInt();
```

Comment:
validate these are valid options for year, month and day, dont let someone insert nonsense like 20/67/45